src/wallet/wallet2.h

```diff
@@ -821,8 +822,6 @@ namespace tools
       a & m_ring_history_saved;
     }
 
-    multisig_tx_set make_multisig_tx_set(const std::vector<pending_tx>& ptx_vector) const;
-
```

Comment:
Please squash this change to the previous commit where this function was introduced.